spanish-terminology.md

```diff
@@ -0,0 +1,32 @@
+﻿# Glossary For Spanish Translations
+
+| **Original** | **Traducción** | **Notas** |
+| --- | --- | --- |
+| Amounts | Cantidades | |
+| Blockchain | Blockchain | Mantiene su forma original al ser una nueva tecnología |
+| Daemon | Daemon |  |
+| Flag | Marcador | Flag es un marcador. En algunas traducciones en la GUI se utiliza la palabra opción |
+| Forum Funding System - FFS | Sistema de Financiamiento Colectivo - SFC | |
+| Hash | Hash | --- |
+| Keys | Claves | Se discutió "llaves", pero la comunidad Bitcoin ya utiliza la palabra "claves" |
+| Miner | Minero | |
+| Mnemonic seed | Semilla mnemónica | |
+| Open-source | Código abierto | |
+| Paper wallet | Monedero en papel | |
+| Pool | Pool | Se queda "pool" y no "piscina", ya que pool hace referencia a esfuerzo o valor compartido |
+| Receiver | Receptor | |
+| Ring signature | Firma de círculo | Se mantiene como firma de círculo por su uso en wikipedia |
+| Sender | Emisor | |
+| Spend Key | Clave de utilización | |
+| Stagenet | Fase de la red | Para discusión. Dejarla como está o traducirla |
+| Tail emission | Emisión final | A discusión porque emisión de "cola" o "trasera" puede mal entenderse |
+| Untraceable | No rastreable | Para slogan se propone "Imposible de rastrear", para lo demás, "no rastreable" |
+| View Key | Clave de visualización | |
+| Wallet | Monedero | Se tomó "monedero" porque ya está siendo utilizado en la comunidad de Bitcoin |
+| White paper | Libro blanco | Wikipedia define el término como Libro blanco |
+
+
+## Not translated
```

Comment:
This section does not display properly for some reason

Reply:
It works now, missed a line for the table template.
BTW how do I quote a specific line?